cluster_management/src/test/java/com/pinterest/rocksplicator/codecs/ThriftCodecTest.java

```diff
@@ -0,0 +1,30 @@
+package com.pinterest.rocksplicator.codecs;
+
+import static org.junit.Assert.assertEquals;
+
+import com.pinterest.rocksplicator.thrift.eventhistory.LeaderEventsHistory;
+
+import org.junit.Test;
+
+public class ThriftCodecTest extends CodecTestBase {
+
+  @Test
+  public void testBinaryCodec() throws Exception {
+    ThriftCodec<LeaderEventsHistory>
+        codec =
+        ThriftCodec.createBinaryCodec(LeaderEventsHistory.class);
+    byte[] binaryData = codec.encode(history);
+    LeaderEventsHistory decodedHistory = codec.decode(binaryData);
+    assertEquals(history, decodedHistory);
```

Comment:
I'm missing this in all the tests, but where is this variable `history` declared?

Reply:
In the CodecTestBase class which is Base class for all codec tests.